performance_tests/src/aws_encryption_sdk_performance_tests/master_key_providers/caching_cmm.py

```diff
@@ -0,0 +1,85 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Performance tests for the Caching Cryptographic Materials Manager (CMM)."""
```

Comment:
We need to note that this is the caching CMM with the KMS MKP.

Ideally we might have considered this in our performance test's input dimensions:
```
key_provider: [kms mkp, raw rsa mkp, raw kms mkp]
cmm: [default, caching]
```

Reply:
Updated in the doc string for the file